### ISSUE TITLE
gateway-rfxrec433-usb fails to load

### ISSUE THREAD

**Issue by Travis-Snoozy** [TheThingSystem/steward#70]
Tl;dr: this module looks completely broken, I can't see how to fix it, it seems reasonable to disable it.

The following prevents the steward from starting up with current git HEAD:

alert: [steward] exception diagnostic=Object #<Object> has no method 'devices'
alert: [steward] exception stack=[{"fileName":"/home/travis.snoozy/src/steward/steward/devices/devices-gateway/gateway-rfxrec433-usb.js","lineNumber":107,"functionName":"scan","typeName":"Object","methodName":null,"columnNumber":10,"native":false},{"fileName":"/home/travis.snoozy/src/steward/steward/devices/devices-gateway/gateway-rfxrec433-usb.js","lineNumber":166,"functionName":"Object.exports.start","typeName":"Object","methodName":"exports.start","columnNumber":3,"native":false},{"fileName":"/home/travis.snoozy/src/steward/steward/core/utility.js","lineNumber":170,"functionName":null,"typeName":null,"methodName":null,"columnNumber":41,"native":false},{"fileName":"fs.js","lineNumber":107,"functionName":"Object.oncomplete","typeName":"Object","methodName":"oncomplete","columnNumber":15,"native":false}]
uncaught exception: TypeError: Object #<Object> has no method 'devices'

I'm currently working around the problem by removing the offending JS file.

The offending chunk of code is:

var scan = function() {
  rfxcom.devices(function(err, results) {

I cannot divine the intent of the caller, otherwise this would be a pull request to fix. Normally I'd fudge versions and do a fixup based on the previous working version... but don't see how this ever worked. Running through a git bisect, it looks like the gateway code has always tried to call this devices function, and the rfxcom code has never provided it. Additionally, I don't see any device enumeration code in the rfxcom package, and the gateway seems to be trying to find "rfxcom:" devices in the global devices list, without any corresponding code that would put such devices in.
